terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1211,6 +1211,34 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                     mounts:
                     - ["/dev/nvme1n1", "/mnt/gitlab", "ext4", ""]
                     rancher:
+                      services:
+                        clamscan:
+                          image: clamav/clamav:0.104
+                          command: |
```

Comment:
```suggestion
                          command: >
```

and eliminate the `\\`

Reply:
I attempted using the format you are suggesting in earlier iterations but ros failed to correctly parse the command. The each option to `clamscan` would be attempted to be run as a command. I'll try it again.

Reply:
It worked 🎉